src/navigation/Routes.android.js

```diff
@@ -180,6 +184,12 @@ function MainNavigator() {
         name={Routes.WELCOME_SCREEN}
         options={{ animationEnabled: false, gestureEnabled: false }}
       />
+      <Stack.Screen
```

Comment:
this fixes an issue where the app would stall and crash when trying to load the secret phrase